### ISSUE TITLE
idea: convert non-pod project structure to pod structure

### ISSUE THREAD

**Issue by shime** [abuiles/ember-watson#52]
it would be great to have  a command that would merge the old project to the new pod structure

this is probably related to #44 and ember-cli/ember-cli#2652

**Comment by knownasilya** [comment]
There is already an existing effort that tries to address that same idea: https://github.com/trabus/ember-cli-podify

**Comment by abuiles** [comment]
Closing in favor of https://github.com/trabus/ember-cli-podify

**Comment by brookjordan** [comment]
Is this still being worked on?

**Comment by fivetanley** [comment]
@brookjordan given abuiles comment above, I would assume not.

**Comment by brookjordan** [comment]
@fivetanley The repo he mentioned at https://github.com/trabus/ember-cli-podify looks kinda dead.

**Comment by fivetanley** [comment]
I can re-open the issue, but it needs a champion. Given the difficulty level and in-flux current state of pods, I'm not sure anyone would have the time to dedicate to it.

**Comment by brookjordan** [comment]
Ah ok. Makes sense.
Pity, I just got all excited when I was searching for a way to convert a big project and this popped up.

**Comment by abuiles** [comment]
I know @trabus wants to complete this but since the addon story is not certain yet we would rather hold until there is something.

I work full-time in a big project where we are using pods as they are today, so once the story is clear I'll be interested in exploring this further ... in the meantime, I won't work on it...  contributions are welcome.

**Comment by trabus** [comment]
Yeah, I hit a wall on ember-cli-podify, which lead to ember-cli-mv and ember-cli-rename, to unblock podify. Unfortunately I hit a wall there as well, specifically updating paths and whatnot (I needed help working with ASTs). There are a lot of difficulties with moving and renaming files, that I believe we will need to add support for in watson to support this work. There is also the issue of parsing hbs files, you currently have to use regexp, which sucks.

Anyhow, I do want to pick this back up, moving the bulk of the work to ember-watson. I just need to find the time. I also had some supporting work to do on structures that should make such transforms between pods and non-pods easier.

**Comment by NullVoxPopuli** [comment]
Any updates on this? I have a quickly growing project that would benefit greatly from pods.

**Comment by knownasilya** [comment]
This probably won't move until the new pods RFC is implemented.
